data/digests/34873b4d2390.json

```diff
@@ -0,0 +1,110 @@
+{
+    "id": "34873b4d2390",
+    "title": "Holding the focus",
+    "impactStatement": "A new open source software tool makes it possible for researchers to view cells in living animals in high detail.",
+    "stage": "preview",
+    "updated": "2018-07-30T14:01:23Z",
+    "image": {
+        "thumbnail": {
+            "uri": "https://iiif.elifesciences.org/medium/1*bYUvErUkNw-bG0WSG4fngA.gif",
+            "alt": "",
+            "source": {
+                "mediaType": "image/jpeg",
+                "uri": "https://cdn-images-1.medium.com/1*bYUvErUkNw-bG0WSG4fngA.gif",
+                "filename": "1 wVZUeULhTTv3FHoa8H4s0Q.jpeg"
+            },
+            "size": {
+                "width": 640,
+                "height": 360
+            }
+        }
+    },
+    "subjects": [
+        {
+            "id": "biochemistry-chemical-biology",
+            "name": "Biochemistry and Chemical Biology"
+        }
+    ],
+    "content": [
+        {
+            "type": "image",
+            "image": {
+                "uri": "https://iiif.elifesciences.org/medium/1*bYUvErUkNw-bG0WSG4fngA.gif",
+                "alt": "",
+                "attribution": [
+                    "Clip taken from Video 2 of <a href=\"https://doi.org/10.7554/eLife.35800\">Warren et al.</a> (<a href=\"https://creativecommons.org/licenses/by/4.0/\">CC-BY 4.0</a>)"
+                ],
+                "source": {
+                    "mediaType": "image/jpeg",
```

Comment:
🤔

Reply:
Given https://github.com/elifesciences/jira-import/issues/4358 I'll use the JPG version for everything (which has a .gif extension)

Reply:
Too confusing, I'll get a dummy image